ops/e2e.Dockerfile

```diff
@@ -147,6 +154,7 @@ RUN update-alternatives --install /usr/local/cuda cuda /usr/local/cuda-11.2 9 \
 # Build base Triton server
 ARG PARALLEL=4
 WORKDIR /src/server-${TRITON_VERSION}/build/output
+RUN patch -ruN -d /usr/include/ < ../libdcgm/dcgm_api_export.patch
```

Comment:
I dont see where the patch file is coming from,  is this added somewhere else?

Reply:
This is part of the Triton repo: https://github.com/triton-inference-server/server/blob/main/build/libdcgm/dcgm_api_export.patch